### ISSUE TITLE
White tabs on issue comment ui

### ISSUE THREAD

**Issue by forivall** [DarkThemeHub/GithubDarkTheme#182]
**Describe the bug**
white tabs on issue comment ui when less than 767px
```css
@media (max-width:767px) {
.tabnav--responsive .tabnav-tab {
    background-color: #fafbfc;
    border: 1px solid #e1e4e8;
    border-left: 0;
    border-radius: 0;
}
}
```

, and white formatting toolbar when width is less than 1012px
```css
.page-responsive .previewable-comment-form .comment-form-head.tabnav .toolbar-commenting {
    background: #fff;
}
@media (min-width:1012px) {
 .page-responsive .previewable-comment-form .comment-form-head.tabnav .toolbar-commenting {
  background:transparent
 }
}
```

**Expected behavior**
should be dark

**Screenshots**
![Screen Shot 2020-07-14 at 12 27 44 PM](https://user-images.githubusercontent.com/760204/87468180-fc4b9200-c5cd-11ea-9aae-5be716118581.png)

![Screen Shot 2020-07-14 at 12 31 36 PM](https://user-images.githubusercontent.com/760204/87468190-feadec00-c5cd-11ea-8669-39d4cb64f218.png)

**If this is a beta feature**
Please paste the html here of the element, including the parent element housing it.

NOTE: beta features are prone to change, and may not be worked on depending how much work is to be done.

**Comment by Snazzie** [comment]
Ahh this problem again.

**Comment by forivall** [comment]
grrr, i switched to a different computer that i haven't used in a while, and stylish *said* it updated the style, but it was actually still on 1.24.0. it's solved in 1.32.0
